thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/jdbc/AnomalyFunctionManagerImpl.java

```diff
@@ -63,4 +64,17 @@ public AnomalyFunctionManagerImpl() {
     }
     return result;
   }
+
+  @Override
+  public AnomalyFunctionDTO findWhereNameEquals(String name) {
```

Comment:
If alert and function name has to be unique, could we set their name field to be unique in the DB. There are two benefits:
1. No accidental insertion of duplicate names even we forgot the manual check. Moreover, we embedded this knowledge in the DB and we won't forget about it.
2. We know there is either empty or one function to be returned. Hence, findWhereNameEquals method does not need to process a list of functions.

Reply:
Yes. They are set to be unique; both function name and alert name.